packages/client/hmi-client/src/types/common.ts

```diff
@@ -79,6 +79,8 @@ export type SidePanelTab = {
 
 // Tabs
 export type Tab = {
-	name: string;
-	props?: Object;
+	label: string;
+	icon: string;
+	assetId?: string | number;
+	assetType?: string; // Change to AssetType later
```

Comment:
Can't the `assetType` be inferred when fetching the _Asset_ in the _Tab_ component?

Reply:
Not sure exactly what you mean. I made `assetType` have a type of `ProjectAssetType` but I am not sure if that's what you meant